tests/integration/test_integration_metobs.py

```diff
@@ -289,7 +265,12 @@ def test_integration_metobs_second(
         assert data.title == data_title
         if table:
             assert data.data.iloc[table_locr, table_locc] == table
-            assert data.raw_data_header == raw_header_0
+            assert data.raw_data_header[0:324] == raw_header_0[0:324]
+            data.data_header.pop("Tidsperiod (t.o.m)")
+            try:
+                header_0.pop("Tidsperiod (t.o.m)")
+            except KeyError:
+                pass
```

Comment:
Perhaps we should at least log the error?

Reply:
Any suggestions as to where and how?

Reply:
```
import logging

logger = logging.getLogger(__name__)



logger.info("...")
```

Reply:
Ok. I am somewhat familiar with logging, but for whom are we storing these logs? This is a somewhat ugly solution to the problem that the variable is somehow persistent between calls. I can only pop the end date once, essentially. So rather than importing logging and going through all of the hassle of setting up a log file or whatever, perhaps what you are really suggesting is a neater formulation, something along the terms of
```
if exist(header0["end date"]):
    header0["end date"].pop()
```
?

Reply:
Changed as per my suggestion in https://github.com/Ingenjorsarbete-For-Klimatet/ifk-smhi/pull/98/commits/2cd1d5367480da60da64ced32cee3947529fbd9d